docs/kubernetes/windows.md

```diff
@@ -1,446 +1,309 @@
+```powershell
+az group deployment create --name plangk8swin1-deploy --resource-group k8s-win1 --template-file ".\_output\plangk8swin1\azuredeploy.json" --parameters ".\_output\plangk8swin1\azuredeploy.parameters.json"
+```
+
+After several minutes, it will return a whole bunch of JSON. Look for `masterFQDN`.
 
-#### make sure storageclass is created successfully
 ```
-kubectl get storageclass/azuredisk -o wide
+      "masterFQDN": {
+        "type": "String",
+        "value": "plangk8swin1.westus2.cloudapp.azure.com"
+      },
 ```
 
-#### 2. Create a pvc for azure disk
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azuredisk/pvc-azuredisk.yaml```
+#### Check that the cluster is up
 
-#### make sure pvc is created successfully
-```
-kubectl get pvc/pvc-azuredisk -o wide
+SSH to that FQDN with the username you put in the linuxProfile section of `kubernetes-windows.json` :
+
+```powershell
+ssh azureuser@plangk8swin1.westus2.cloudapp.azure.com
 ```
 
-#### 3. Create a pod with azure disk pvc
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azuredisk/iis-azuredisk.yaml```
+Now, you can verify the cluster is up with `kubectl get node -o wide`. Once you've checked that the cluster is up, go ahead and disconnect from the Linux master by running `exit`
 
-#### watch the status of pod until its `STATUS` is `Running`
+```bash
+azureuser@k8s-master-40336153-0:~$ kubectl get node -o wide
+NAME                    STATUS    ROLES     AGE       VERSION   EXTERNAL-IP   OS-IMAGE                    KERNEL-VERSION   CONTAINER-RUNTIME
+40336k8s9000            Ready     <none>    21m       v1.9.10   <none>        Windows Server Datacenter   10.0.17134.112
+                        docker://17.6.2
+40336k8s9001            Ready     <none>    20m       v1.9.10   <none>    Windows Server Datacenter   10.0.17134.112
+                        docker://17.6.2
+k8s-master-40336153-0   Ready     master    22m       v1.9.10   <none>    Ubuntu 16.04.5 LTS   4.15.0-1018-azure   docker://1.13.1
 ```
-watch kubectl get po/iis-azuredisk -o wide
+
+
+Next, copy the configuration file needed to manage the cluster with your Windows machine using `scp`, then set `$ENV:KUBECONFIG` to the path of that file.
+
+```powershell
+scp azureuser@plangk8swin1.westus2.cloudapp.azure.com:~/.kube/config plangk8swin1
+$ENV:KUBECONFIG = (Get-Item .\plangk8swin1).FullName
 ```
 
-#### 4. Enter the pod container to validate
+#### Get Kubectl for Windows
+
+The latest release of Kubernetes Control (kubectl) is available on the [Kubernetes release page](https://kubernetes.io/docs/imported/release/notes/). Look for `kubernetes-client-windows-amd64.tar.gz` and download it.
+
+Windows 10 version 1803 already includes `tar`, so extract the archive and move `kubectl.exe` to your local path
+
+```powershell
+tar xvzf C:\Users\patrick\Downloads\kubernetes-client-windows-amd64.tar.tar

```

Comment:
.tar.tar?

Reply:
That's not a typo. That's reflecting what Edge actually named the file. I'll go ahead and change it to .tar.gz to be less confusing and hope people can figure it out :)